test.sh

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env bash
 if ! [ -n "$1" ] || ! [ -n "$2" ]; then
-	echo "USAGE: ./test.sh PROGRAM TEST_SUITE"
+	echo "USAGE: ./test.sh BINARY TEST_SUITE"
 	exit 1
 fi
 
-if ! dune build; then
-	exit 1
-fi
+# if ! dune build; then
+# 	exit 1
+# fi
 
 export DBL_LIB="lib/"
 
 TIMEOUT=1.0
 
-binary="_build/default/src/$1.exe"
+binary=$1
```

Comment:
I see a minor problem with this change. When I run tests locally, I would like to use a version of dbl compiled from source on the active branch, not the one that is installed on my system.

Reply:
A bit of an oversight on my part - in my dev environment, `dbl` in `PATH` is the one from `_build/...`. To be fair, you still can run `./test.sh ./_build/default/src/dbl.exe ./test/test_suite` to test against your freshly built `dbl`, but I understand that the default workflow should probably remain unchanged.

Reply:
I restored most of the original behaviour of `test.sh` in my branch - when passed a relative path as `PROGRAM` it does what it used to do.